paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/viewmodel/PaymentSheetPlaygroundViewModel.kt

```diff
@@ -33,6 +36,34 @@ internal class PaymentSheetPlaygroundViewModel(
     var checkoutMode: CheckoutMode? = null
     var temporaryCustomerId: String? = null
 
+    private val sharedPreferencesName = "playgroundToggles"
+
+    fun setSharedPreferences(customer: String, googlePay: Boolean, currency: String, mode: String, setShippingAddress: Boolean, setAutomaticPaymentMethods: Boolean) {
+        val sharedPreferences = getApplication<Application>().getSharedPreferences(sharedPreferencesName, AppCompatActivity.MODE_PRIVATE)
+        val editor = sharedPreferences.edit()
+
+        editor.putString(Toggle.Customer.key, customer)
```

Comment:
thinking out loud here: Is there way we could stream or loop through your toggle class and not have to add a new line for every toggle? It would be cool if I could just add an enum and these methods would handle the logic of fetching and retrieving for me. 

If you added a third field to the enum for the type we'd know what method to use for shared preferences. What do you think? I'm just trying to reduce the logical load to add a toggle that gets restored.

Reply:
Wouldn't we still have to add a new line for each toggle though, because we need to specify which variable to store. E.g. We need to specify to use the 'customer' variable for the customer toggle

Reply:
hmmm good point. We can think a little more. I'm just trying to see if we can reduce the amount of changes needed to do when adding a toggle.